src/openforms/appointments/api/serializers.py

```diff
@@ -59,6 +59,18 @@ class Meta:
         ref_name = "AppointmentProduct"
 
 
+class ProductInputSerializer(serializers.Serializer):
+    product_id = serializers.ListField(
```

Comment:
this is a bit of an odd one because we are abusing the serializer for query string params, and that would make it look like `?product_ids=123&product_ids=456` which is inconsistent with the rest of the endpoints that use `?product_id=123&product_id=456`